distributions/aws/examples/rds-s3/README-automated.md

```diff
@@ -0,0 +1,174 @@
+# Kustomize Manifests for RDS and S3
+
+## Overview
+
+This Kustomize Manifest can be used to deploy Kubeflow Pipelines and Katib with RDS and S3.
+
+Follow the [install](#install) steps below to configure and deploy the Kustomize manifest.
+
+## Install
+
+### 1. Prerequisites
+
+1. Install CLI tools
+
+   - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) - A command line tool for interacting with AWS services.
+   - [eksctl](https://eksctl.io/introduction/#installation) - A command line tool for working with EKS clusters.
+   - [kubectl](https://kubernetes.io/docs/tasks/tools) - A command line tool for working with Kubernetes clusters.
+   - [yq](https://mikefarah.gitbook.io/yq) - A command line tool for YAML processing. (For Linux environments, use the [wget plain binary installation](https://mikefarah.gitbook.io/yq/#wget))
+   - [jq](https://stedolan.github.io/jq/download/) - A command line tool for processing JSON.
+   - [kustomize version 3.2.0](https://github.com/kubernetes-sigs/kustomize/releases/tag/v3.2.0) - A command line tool to customize Kubernetes objects through a kustomization file.
+     - :warning: Kubeflow 1.3.0 is not compatible with the latest versions of of kustomize 4.x. This is due to changes in the order resources are sorted and printed. Please see [kubernetes-sigs/kustomize#3794](https://github.com/kubernetes-sigs/kustomize/issues/3794) and [kubeflow/manifests#1797](https://github.com/kubeflow/manifests/issues/1797). We know this is not ideal and are working with the upstream kustomize team to add support for the latest versions of kustomize as soon as we can.
+
+2. Clone the repo and checkout the release branch
+
+```
+git clone https://github.com/awslabs/kubeflow-manifests.git
+cd kubeflow-manifests
+git checkout v1.3-branch
+```
+
+3. Create an EKS cluster
+
+Run this command to create an EKS cluster by changing `<YOUR_CLUSTER_NAME>` and `<YOUR_CLUSTER_REGION>` to your preferred settings. More details about cluster creation via `eksctl` can be found [here](https://eksctl.io/usage/creating-and-managing-clusters/).
+
+```
+export CLUSTER_NAME=<YOUR_CLUSTER_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+
+eksctl create cluster \
+--name ${CLUSTER_NAME} \
+--version 1.19 \
+--region ${CLUSTER_REGION} \
+--nodegroup-name linux-nodes \
+--node-type m5.xlarge \
+--nodes 5 \
+--nodes-min 1 \
+--nodes-max 10 \
+--managed
+```
+
+### 2. Setup RDS & S3
+```
+export CLUSTER_NAME=<YOUR_CLUSTER_NAME>
+export CLUSTER_REGION=<YOUR_CLUSTER_REGION>
+export S3_BUCKET=<YOUR_S3_BUCKET_NAME>
+chmod 755 auto-rds-s3-setup.sh
+./auto-rds-s3-setup.sh
+```
```

Comment:
Can we just add this section and mentioned you can skip steps (4,5, 6 and 7)  into existing README.md ?

Reply:
Sure thing, I deleted the `README-automated.md`, moved the part you mentioned inside the existing `README.md` and updated it to make a clear distinction between the automated and manual setup.  
   
Let me know what you think